.github/workflows/buildandtest.yml

```diff
@@ -97,112 +77,99 @@ jobs:
     if: ${{ needs.diff_check.outputs.skip_ci != 'true' }}
     steps:
       - uses: actions/checkout@v4
+      - run: corepack enable
       - uses: actions/setup-node@v4
         with:
           node-version: 18
-      - uses: actions/cache@v4
-        id: cache
-        with:
-          path: node_modules
-          key: ${{ runner.os }}-${{ github.sha }}
+          cache: 'yarn'
+          cache-dependency-path: yarn.lock
       - name: Install Dependencies
-        if: ${{ steps.cache.outputs['cache-hit'] != 'true' }}
         run: yarn install
       - name: Build
         run: yarn build
       - name: Archive dist
         uses: actions/upload-artifact@v4
         with:
           name: dist
-          path: dist
+          path: packages/core/dist
       - name: Archive ts3.8
         uses: actions/upload-artifact@v4
         with:
           name: ts3.8
-          path: ts3.8
+          path: packages/core/ts3.8
       - name: Archive Expo Plugin
         uses: actions/upload-artifact@v4
         with:
           name: expo-plugin
-          path: plugin/build
+          path: packages/core/plugin/build
       - name: Pack
-        run: yarn pack
+        run: |
+          cp README.md packages/core
```

Comment:
l (no action required):
Are you keeping the readme of the SDK top level and you copy it to the package to get it into the tarball? No objections btw, just curious.

In the JS SDK monorepo we keep a Readme per package but I guess that's only gonna become necessary if you have more than one package.